pkgs/development/python-modules/quickfix/default.nix

```
@@ -0,0 +1,19 @@
+{ lib, stdenv, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "quickfix";
+  version = "1.15.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-41ZK0t0VWJLcdEltb02xeUvY+fdEUMy7RVcf+qKBXaI=";
+  };
+
+  meta = with lib; {
+    description = "FIX Engine Library";
+    homepage = "https://github.com/quickfix/quickfix/";
+    license = licenses.free;
```

We can give more descriptive license.
```suggestion
    license = {
      fullName = "The QuickFIX Software License, Version 1.0";
      free = true;
      redistributable = true;
      url = "https://github.com/quickfix/quickfix/blob/master/LICENSE";
    };
```